config/default.json

```diff
@@ -32,7 +33,12 @@
     "apiKey": "R7fyLMRrR2IF1t265ShCQISh9BVhM8Yw"
   },
   "canvasPoller": {
-    "enabled": true,
+    "enabled": false,
```

Comment:
I think the tests already disable the poller, so this can probably remain enabled by default?